### PR TITLE
Fix for dumb or unset TERM

### PR DESCRIPTION
**Pull request by tdaitx**
This is a fix for when `$TERM` is unset or set to _dumb_ - eg. the _dumb_ value is used by _scp_. For some reason `clear` complains that the TERM variable is not properly set (it is) and causes an error:

```
$ scp <file> root@dietpi
TERM environment variable not set.
 --------------------------------
```

With the fix I am able to properly copy the file. I'm using the openssh-server package, I haven't tested it with the default dropbear.

Note that only exporting `TERM` is **not** enough, all output should be disabled otherwise scp will fail.

Reference:
[export TERM=${TERM:-dumb}](http://stackoverflow.com/questions/19425727/how-to-remove-term-environment-variable-not-set)

### CODE REVIEW

File: dietpi/dietpi-banner [hunk 1]
```diff
@@ -56,6 +56,11 @@
 	#/////////////////////////////////////////////////////////////////////////////////////
 	# Main Loop
 	#/////////////////////////////////////////////////////////////////////////////////////
+	export TERM=${TERM:-dumb}
+	if (( "$TERM" == "dumb" )); then
```
Comment (Fourdee) [inline dietpi/dietpi-banner:60]: exit also gets triggered with dropbear logins.

Heres the value of TERM:
export TERM=${TERM:-dumb}; echo -e "$TERM"
xterm

I tested with the following, exit doesnt get triggered with dropbear:
if [ "$TERM" = "dumb" ]; then

Tested with the following to confirm trigger works (exit got triggered):
if [ "$TERM" = "xterm" ]; then

If you can test with openssh server, just to verify:
if [ "$TERM" = "dumb" ]; then

Thanks tdaitx.

Comment (tdaitx) [inline dietpi/dietpi-banner:60]: Yes, exit is triggered on _scp_ with openssh server and **not** triggered on the usual _ssh_ login (just it should be). Thanks!

Comment (Fourdee) [inline dietpi/dietpi-banner]: Thanks tdaitx,

I'll add this to the master branch manually. I'am unable to change/modify the code in this request as i dont have access to "patch-2".

**Comment by Fourdee** [comment]
Thanks tdaitx,

I'll add this to the master branch manually. I'am unable to change/modify the code in this request as i dont have access to "patch-2".

Also moved to the login script line 39. This should prevent any further SCP connection issues down the road. 
https://github.com/Fourdee/DietPi/blob/master/dietpi/login

**Pull request closed without merging**